runtime/tutor

```diff
@@ -519,7 +519,7 @@ _________________________________________________________________
     like 'oranges'.
  6. Type , to remove the second cursor.
 
- --> I like to eat apples since my favorite fruit is apples.
+ --> I like to eat apples since my favourite fruit is apples.
      I like to eat oranges since my favourite fruit is oranges.
```

Comment:
actually I think the typo is the one with the `u` - there's another `favorite` down on L833

Reply:
https://www.grammarly.com/blog/favorite-favourite/

`favorite` is American English, it is `favourite` everywhere else.

Reply:
It seems like we use American English elsewhere in the tutor, but the other "favorite" is the only example I found of that. I don't have strong feelings either way between American vs. British

Reply:
so i assume i will fix it up to use the AE version.